tests/template_tests/syntax_tests/test_lorem.py

```diff
@@ -18,3 +19,24 @@ def test_lorem_random(self):
         self.assertEqual(len(words), 3)
         for word in words:
             self.assertIn(word, WORDS)
+
+    @setup({'lorem_default': '{% lorem %}'})
+    def test_lorem_default(self):
+        output = self.engine.render_to_string('lorem_default')
+        self.assertEqual(output, COMMON_P)
+
+    @setup({'lorem_syntax_error': '{% lorem 1 2 3 4 %}'})
+    def test_lorem_syntax(self):
+        msg = "Incorrect format for 'lorem' tag"
+        with self.assertRaisesMessage(TemplateSyntaxError, msg):
+            self.engine.render_to_string('lorem_syntax_error')
+
+    @setup({'lorem_multiple_paragraphs': '{% lorem 2 p %}'})
+    def test_lorem_multiple_paragraphs(self):
+        output = self.engine.render_to_string('lorem_multiple_paragraphs')
+        assert output.count("<p>") == 2
```

Comment:
I think there may be something better here. I tried `self.assertInHtml` but couldn't get this to work, the challenge is the 2nd  `<p>` contains random words. I therefore resorted to a standard assert and tested that two paragraphs are rendered.

Reply:
Use `self.assertEqual()`.

Reply:
Oh, yes. We're not using pytest either. 👍